src/common/paths/dbrps_dbrpID.yml

```
@@ -56,77 +92,135 @@ patch:
   responses:
     "200":
       description: An updated mapping
       content:
         application/json:
           schema:
             $ref: "../schemas/DBRPGet.yml"
-    "404":
-      description: The mapping was not found
-      content:
-        application/json:
-          schema:
-            $ref: "../schemas/Error.yml"
+          examples:
+            $ref: "../responses/example-responses/PatchDBRPIDSuccessResponse.yml"
     "400":
-      description: if any of the IDs passed is invalid
+      description: Bad Request. Query parameters contain invalid values.
       content:
         application/json:
           schema:
             $ref: "../schemas/Error.yml"
+          examples:
+            invalidRequest:
+              description: |
+                The query parameters contain invalid values.
+              value:
+                {
+                  "code": "invalid",
+                  "message": "invalid ID"
+                }
+    "401":
+      $ref: "../responses/AuthorizationError.yml"
+    "404":
+      $ref: "../responses/ResourceNotFoundError.yml"
+    "500":
+      $ref: "../responses/InternalServerError.yml"
     default:
       description: Unexpected error
       content:
         application/json:
           schema:
             $ref: "../schemas/Error.yml"
+  x-codeSamples:
+    - lang: Shell
+      label: cURL
+      source: |
+        curl --request PATCH \
+           "http://localhost:8086/api/v2/dbrp/DBRP_ID" \
+          --header 'Content-type: application/json' \
+          --header "Authorization: Token INFLUXDB_TOKEN" \
+          --data-binary @- << EOF
+            { \
+              "default": true, \
+              "retention_policy": "example_retention_policy_name" \
+            }
+        EOF
 delete:
   operationId: DeleteDBRPID
   tags:
     - DBRPs
   summary: Delete a database retention policy
+  description: |
+    Deletes the specified database retention policy (DBRP) mapping.
+
+    #### Related guide
+
+    - [Database and retention policy mapping]({{% INFLUXDB_DOCS_URL %}}/reference/api/influxdb-1x/dbrp/)
   parameters:
     - $ref: "../parameters/TraceSpan.yml"
     - in: query
       name: orgID
-      description: Specifies the organization ID of the mapping
+      description: |
+        An organization ID.
+        Only returns the specified organization.
       schema:
         type: string
     - in: query
       name: org
-      description: Specifies the organization name of the mapping
+      description: |
+        An organization name.
+        Only returns the specified organization.
       schema:
         type: string
     - in: path
       name: dbrpID
       schema:
         type: string
       required: true
-      description: The database retention policy mapping
+      description: |
+        A DBRP mapping ID.
+        Only returns the specified DBRP mapping.
   responses:
     "204":
       description: Delete has been accepted
```

```suggestion
      description: Success. The delete is accepted.
```

Or one of these (not sure which is true):
 - "The DBRP mapping is deleted."
 - "The DBRP mapping is queued for deletion."